src/main/java/wooteco/subway/admin/domain/Station.java

```diff
@@ -10,6 +10,15 @@
     private String name;
     private LocalDateTime createdAt;
 
+    public Station() {
+    }
+
+    public Station(Long id, String name) {
+        this.id = id;
+        this.name = name;
+        this.createdAt = LocalDateTime.now();
+    }
+
     public Station(String name) {
         this.name = name;
         this.createdAt = LocalDateTime.now();
```

Comment:
```suggestion
        this(null, name);
```

객체의 초기화는 하나의 생성자에서 해주는 것이 좋습니다.
객체의 초기화 또한 중복이라는 것을 명심해주세요!